Library/Homebrew/cask/cask.rb

```diff
@@ -380,10 +381,22 @@ def to_hash_with_variations
         end
       end
 
-      hash["variations"] = variations
+      hash["variations"] = @variations_hash
       hash
     end
 
+    def to_api_hash
+      api_hash = to_hash_with_variations.except(
+        # Included in the top-level of the `internal_cask_v3.json` file
+        # created by the `brew generate-cask-api` command.
+        "old_tokens",
```

Comment:
- Unless I'm mistaken: `token` and `full_token` are always identical, too.
- `url_specs`, `languages` seem to be often empty. 
- stuff like `appcast` is often `null`
- `installed` and `installed_time` and `outdated` are reflective of the machine they are run on and not public API.

Reply:
`appcast` is deprecated, if it's not `null` it should be.